CRM/CiviGeometry/BAO/Geometry.php

```diff
@@ -265,8 +265,63 @@ public function calculateDistance($params) {
       1 => [$params['geometry_a'], 'String'],
       2 => [$params['geometry_b'], 'String'],
     ]);
-    $meters = (float) $result * 1000;
-    return $meters;
+    $metres = (float) $result * 1000;
+    return $metres;
+  }
+
```

Comment:
I think this needs a comment to say exactly what kind of spatial information you're returning.

Why is it all returning in a single function as opposed to using multiple functions, one for each piece of information?